elementor.php

```diff
@@ -77,8 +77,15 @@ function elementor_load_plugin_textdomain() {
  * @return void
  */
 function elementor_fail_php_version() {
-	/* translators: %s: PHP version. */
-	$message = sprintf( esc_html__( 'Elementor requires PHP version %s+, plugin is currently NOT RUNNING.', 'elementor' ), '7.0' );
+	$message = sprintf(
+	/* translators: 1: Bold text opening tag, 2: Bold text closing tag, 3: translators: %s: PHP version. 4: Link opening tag, 5: Link closing tag. */
+	esc_html__( '%1$sElementor isn’t running because PHP is outdated.%2$s Update to PHP version %3$s and get back to creating! %4$sShow me how%5$s', 'elementor' ),
```

Comment:
```suggestion
		esc_html__( '%1$sElementor isn’t running because PHP is outdated.%2$s Update to PHP version %3$s and get back to creating! %4$sShow me how%5$s', 'elementor' ),
```